distributed/distributed-schema.yaml

```diff
@@ -770,6 +770,12 @@ properties:
         - "null"
         description:
           The size of the memory pool in bytes
+      logging:
+        type:
+        - bool
```

Comment:
I think the errors are related to the schema checking. 
```suggestion
        - boolean
```

Reply:
Thanks! :)